warrior/Warrior

```diff
@@ -48,6 +49,77 @@ def update_jira_by_id(jiraproj, jiraid, exec_dir, status):
     else:
         print_info("jiraid not provided, will not update jira issue")
 
+def avg_time_calculate(abs_filepath, default_repo):
+    """ To print the Estimated Runtime for the Project/Suite/Testcase is
+    calculated using the previous junit file runtime values """
+
+    fname = abs_filepath.split('/')[-1].split('.')[0]
+    junit_fname = fname + "_junit.xml"
+    file_type = Utils.xml_Utils.getRoot(abs_filepath).tag
+    total = count = 0
+
+    if not default_repo.has_key('ow_resultdir'):
```

Comment:
Too many if conditions with no comments. Add comment to each if to explain your logic